### PR TITLE
fix: Removing the ability to upload documents

### PR DESCRIPTION
**Pull request by matt-crowson-moj**
The ability to upload documents for moves is no longer required.

Historic moves with documents attached can still be downloaded.

**Move with document added (prior to change, as in added move with document attached then took out ability to add):**

image

**New move (without a previously added document):**

image

### CODE REVIEW

File: locales/en/moves.json [hunk 1]
```diff
@@ -101,7 +98,7 @@
     "assessment_answer": {
       "label": "{{question}} details"
     },
-    "documents": {
+    "z": {
```
Comment (thomasleese) [inline locales/en/moves.json:101]: Is this change intentional?

Comment (matt-crowson-moj) [reply locales/en/moves.json:101]: Nope.  I must have hit a key or something ...

**Review by thomasleese — COMMENTED** [review]
Code changes look good, I think it would be worth rebasing (to get rid of the merge commit) and squashing the two other commits together so we don't need the typo fix commit.

**Review by thomasleese — APPROVED** [review]
Looks good!

**Review by ElizabethGoddard — APPROVED** [review]
Awesome, thanks Matt!
